File: src/Usuario.py
import random

class Usuario:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.id = random.randint(1, 1000)
        self.edad = edad
        self.compradoBook = False

    def get_nombre(self):
        return self.nombre

    def get_edad(self):
        return self.edad

    def get_comprado_book(self):
        return self.compradoBook

    def set_comprado_book(self, comprado):
        self.compradoBook = comprado

File: src/Usuario.java
import java.util.ArrayList;
import java.util.Random;

public class Usuario{
    private String nombre;
    private int id;
    private int edad;
    private boolean CompradoBook;
    Random ran = new Random();

    public Usuario(String nombre,int edad){
        this.nombre = nombre;
        this.edad = edad;
        this.id = ran.nextInt(1000)+1 ;
        this.CompradoBook=false;
    }

    public Usuario() {

    }

    public boolean getCompradoBook() {
        return CompradoBook;
    }
    public void setCompradoBook(boolean CompradoBook) {
       this.CompradoBook=CompradoBook;
    }

    public String getNombre() {
        return nombre;
    }

    public int getEdad() {
        return edad;
    }

}
